package.json

```diff
@@ -7,8 +7,8 @@
     "test": "test"
   },
   "scripts": {
-    "test": "grunt test",
-    "build": "bower install; grunt build"
+    "build": "bower install; grunt build",
+    "test": "./node_modules/loopback-testing/node_modules/.bin/mocha common/models/test"
```

Comment:
@bajtos is there a specific way to get grunt to run a command line or should I look at adding [grunt-mocha](https://www.npmjs.com/package/grunt-mocha) or [grunt-exec](https://www.npmjs.com/package/grunt-exec)?